tests/match/test_match.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+import numpy as np
+import math
+from graspy.match import FastApproximateQAP as FAQ
+from graspy.match import SinkhornKnopp as SK
+
+
+class TestFAQ:
```

Comment:
@asaadeldin11  need checks to ensure that if improper matrix shapes are input, raises errors. basically like the input checks for kwargs but for the arguments to fit instead.

basically like the kwarg inputs